.github/workflows/commit-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Commit Tests
+
+env:
+  release_major: 8
+
+on:
+  push:
+
+jobs:
+  build_llvm:
+    name: check-all llvm/clang/pstore
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macOS-latest
+    steps:
+    - name: Install Ninja
+      uses: llvm/actions/install-ninja@master
+    - name: Checkout LLVM
+      uses: actions/checkout@v2
+    - name: Checkout pstore
+      uses: actions/checkout@v2
+      with:
+        repository: SNSystems/pstore
+        path: pstore
+    - name: Test LLVM
+      uses: llvm/actions/build-test-llvm-project@master
+      with:
+        cmake_args: -G Ninja -DLLVM_ENABLE_PROJECTS="clang;pstore" -DLLVM_TARGETS_TO_BUILD=X86 -DLLVM_TOOL_CLANG_TOOLS_EXTRA_BUILD=OFF
```

Comment:
Given that memory consumption is a concern would it be worth limiting the number of parallel links?

Reply:
Maybe it is. It seems to be stable at the moment but it's probably only a matter of time until several big things get linked at the same time. I'll add -DLLVM_PARALLEL_LINK_JOBS=1. Generally this doesn't add much to build time.